### ISSUE TITLE
Multiple deploy.prototxt files

### ISSUE THREAD

**Issue by sanjanajainsj** [qiexing/face-landmark-localization#22]
Hello,
Thank you for the model and codes. I am a bit confused. There are two deploy files provided. The model/deploy.prototxt consists of a model which gets divided at the fully connected layer fc7 to two parts: 68-point and poselayer. However, the train/deploy.prototxt consists of a model which gets divided after the relu4 layer to be passed onto conv5 and conv5_b, and so is the train/train_val.prototxt. What are the differences in results for these and which one do you recommend? Also, we just have one train_val.prototxt. Is the caffemodel trained on a file similar to model/deploy.prototxt? 
Thank you in advance.

**Comment by qiexing** [comment]
Yes, you are right. These are two different network structures. The difference is how many layers these two tasks are shared.  The latter network structure will cost more feed forward time.

**Comment by zhouxiaoxu** [comment]
qiexing 你好，你提供的model是使用 train/train_val.prototxt训练的吗？如果是，那使用 train/train_val.prototx训练的模型可以在使用model/deploy.prototxt做预测吗？
